.github/workflows/pr-image.yaml

```
@@ -0,0 +1,121 @@
+name: Publish BotKube image
+
+on:
+  pull_request_target:
+    types: [ opened, synchronize, reopened ]
+    branches:
+      - "develop"
+    paths-ignore:
+      - 'branding/**'
+      - 'design/**'
+      - 'helm/**'
+      - '.codespell-whitelist'
+      - '.gitignore'
+      - '.golangci.yml'
+      - '.mergify.yml'
+      - 'botkube-title.jpg'
+      - 'botkube_arch.jpg'
+      - 'CHANGELOG.md'
+      - 'CODE_OF_CONDUCT.md'
+      - 'comm_config.yaml'
+      - 'CONTRIBUTING.md'
+      - 'deploy-all-in-one.yaml'
+      - 'deploy-all-in-one-tls.yaml'
+      - 'LICENSE'
+      - 'README.md'
+      - 'resource_config.yaml'
+
+env:
+  GO_VERSION: 1.18
+  PR_NUMBER: ${{ github.event.pull_request.number }}
+
+jobs:
+
+  save-image:
+    name: Build and save BotKube image
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          persist-credentials: false
+
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ env.GO_VERSION }}
+          cache: true
+
+      - name: Install GoReleaser
+        uses: goreleaser/goreleaser-action@v2
+        with:
+          install-only: true
+          version: latest
+
+      - name: Save
+        run: |
+          make save-pr-image
+
+      - name: Upload artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ matrix.APP }}-${{github.sha}}
```

There are still 2 occurrences of `matrix.APP` - here:
```suggestion
          name: botkube-${{github.sha}}
```